Client/core/Graphics/CPrimitive3DBatcher.h

```diff
@@ -0,0 +1,36 @@
+/*****************************************************************************
+ *
+ *  PROJECT:     Multi Theft Auto
+ *  LICENSE:     See LICENSE in the top level directory
+ *  FILE:        CPrimitive3DBatcher.h
+ *  PURPOSE:
+ *
+ *
+ *****************************************************************************/
+// Vertex type used by the primitives 3d batcher
+struct sPrimitive3D
+{
+    D3DPRIMITIVETYPE                 eType;
+    std::vector<PrimitiveVertice>* pVecVertices;
+};
+
+//
+// Batches primitives drawing
+//
+class CPrimitive3DBatcher
+{
+public:
+    ZERO_ON_NEW
+    CPrimitive3DBatcher(bool bPreGUI);
+    ~CPrimitive3DBatcher(void);
```

Comment:
Remove the destructor from header and source file if you don't do any work here.